leadbutt.py

```diff
@@ -85,6 +87,170 @@ def get_options(config_options, local_options, cli_options):
+            "steal": "Percent",
+            "nice": "Percent",
+        },
+        "loadAverageMinute": {
+            "fifteen": "Count",
+            "five": "Count",
+            "one": "Count",
+        },
+        "memory": {
+            "writeback": "Kilobytes",
+            "hugePagesFree": "Count",
+            "hugePagesRsvd": "Count",
+            "hugePagesSurp": "Count",
+            "hugePagesTotal": "Count",
+            "cached": "Kilobytes",
+            "hugePagesSize": "Kilobytes",
+            "pageTables": "Kilobytes",
+            "dirty": "Kilobytes",
+            "mapped": "Kilobytes",
+            "active": "Kilobytes",
+            "total": "Kilobytes",
+            "slab": "Kilobytes",
+            "buffers": "Kilobytes",
+        },
+        "tasks": {
+            "sleeping": "Count",
+            "zombie": "Count",
+            "running": "Count",
+            "stopped": "Count",
+            "total": "Count",
+            "blocked": "Count"
+        },
+        "swap": {
+            "cached": "Kilobytes",
+            "total": "Kilobytes",
+            "free": "Kilobytes"
+        },
+        "network":
+        {
+            "rx": "Bytes/Second",
+            "tx": "Bytes/Second"
+        },
+        "diskIO":
+        {
+            "writeKbPS": "Kilobytes/Second",
+            "readIOsPS": "IO/Second",
+            "await": "Milliseconds",
+            "readKbPS": "Kilobytes/Second",
+            "rrqmPS": "IO/Second",
+            "util": "Percent",
+            "avgQueueLen": "Milliseconds",
+            "tps": "Count",
+            "readKb": "Kilobytes",
+            "writeKb": "Kilobytes",
+            "avgReqSz": "Kilobytes",
+            "wrqmPS": "IO/Second",
+            "writeIOsPS": "IO/Second"
+        },
+        "fileSys":
+        {
+            "used": "Kilobytes",
+            "usedFiles": "Count",
+            "usedFilePercent": "Percent",
+            "maxFiles": "Count",
+            "total": "Kilobytes",
+            "usedPercent": "Percent"
+        },
+        "processList":
+        {
+            "vss": "Kilobytes",
+            "tgid": "Count",
+            "parentID": "Count",
+            "memoryUsedPc": "Percent",
+            "cpuUsedPc": "Percent",
+            "id": "Count",
+            "rss": "Kilobytes"
+        }
+    }
+    return unit_map[category].get(statistic, 'Count')
+
+
+def output_log_results(formatter, ingestion_time, context, value):
+    metric_name = (formatter % context).replace('/', '.').lower()
+    line = '{0} {1} {2}\n'.format(
+        metric_name,
+        value,
+        ingestion_time,
+    )
+    sys.stdout.write(line)
+
+
+def process_log_results(results, options):
+    """
+    Output CW enhanced Monitoring to stdout.
+
+    http://boto.cloudhackers.com/en/latest/ref/logs.html
+    """
+
+    context = {}
+    # iterate over each result
+    for result in results:
+        # timestamp
+        timestamp = result['timestamp'] / 1000
```

Comment:
Either create a constant that says `MILLISECOND_TO_SECOND` or write a comment on the unit of timestamp here. And if it is just a timestamp, why convert it?